src/lib/y2firewall/dialogs/main.rb

```diff
@@ -130,6 +130,13 @@ def swap_api
       # modifications
       def apply_changes
         return false if Yast::Mode.config
+        # Firewall settings will be written into the permanent configurations only.
+        # So the running firewalld service will not be changed. Even a reload does
+        # not help (see man pages). So the running firewalld service has to be
```

Comment:
Well, the man is talking about the Runtime configuration, isn't it?, so writing to runtime and doing a reload will lost the changes but doing it in the permanent configuration and doing a reload should work fine.

Reply:
Permanent configuration
       The permanent configuration is stored in config files and will be
       loaded and become new runtime configuration with every machine boot or
       service reload/restart.

Reply:
It depends. While writing the settings we are using --permanent. So the settings will be written to permanent  configuration only. After restarting the service it will affect the running configuration.